index.html

```diff
@@ -1,177 +1,264 @@
+        <a href="https://github.com/aubinleyoung"><img src="assets/icons/1.png" alt="Github link" /></a>
+      </li>
+      <li>
+        <a href="https://www.linkedin.com/in/aubin-simpeze-7a5b7a220/"><img src="assets/icons/2.png"
+            alt="LinkedIn link" /></a>
+      </li>
+      <li>
+        <a href=""><img src="assets/icons/3.png" alt="Angelist link" /></a>
+      </li>
+      <li>
+        <a href="https://twitter.com/SimpezeAubin"><img src="assets/icons/4.png" alt="Twitter link" /></a>
+      </li>
+      <li>
+        <a href=""><img src="assets/icons/5.png" alt="Medium-link" /></a>
+      </li>
+    </ul>
   </section>
   <section id="works-container">
-  <div class="recent-works-title-block">
-  <h2 class="recent-works-title">My Recent Works</h2>
-  <hr class="line">
-  </div>
-  <div class="recent-works">
-  <img src="./assets/images/bgi1.png" alt="" class="work-img">
-  <div class="recent-work-second-block">
-  <h3 class="works-title blue-1">Multi-Post Stories</h3>
-  <p class=" font-montesori blue-2">A daily selection of privately personalized reads; no accounts or sign-upsrequired. has been the industry's standard dummy text ever since the 1500s, when an unknown printer tooka standard dummy text.</p>
-  <ul class="tools-container">
-  <li class="tools">html</li>
-  <li class="tools">css</li>
-  <li class="tools">bootstrap</li>
-  <li class="tools">Ruby</li>
-  </ul>
-  <button class="see-project-btn" type="button">See Project</button>
-  </div>
-  </div>
-  <div class="works work1 ">
-  <h3 class="works-title hide ">Profesional Art <br>Printing Data</h3>
-  <p class="font-montesori hide">A daily selection of privately personalized reads; no accounts or sign-upsrequired. has been the industry's standard dummy text ever since the 1500s, when an unknown printer tooka standard dummy text.</p>
-  <ul class="tools-container hide">
-  <li class="tools bg-btn">html</li>
-  <li class="tools bg-btn">css</li>
-  <li class="tools bg-btn">bootstrap</li>
-  <li class="tools bg-btn">Ruby</li>
-  </ul>
+    <div class="recent-works-title-block">
+      <h2 class="recent-works-title">My Recent Works</h2>
+      <hr class="line" />
+    </div>
+    <div class="recent-works">
+      <img src="./assets/images/bgi1.png" alt="Multi post story" class="work-img" />
+      <div class="recent-work-second-block">
+        <h3 class="works-title blue-1">Multi-Post Stories</h3>
+        <p class="font-montesori blue-2">
+          A daily selection of privately personalized reads; no accounts or
+          sign-upsrequired. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer tooka standard dummy text.
+        </p>
+        <ul class="tools-container">
+          <li class="tools">html</li>
+          <li class="tools">css</li>
+          <li class="tools">bootstrap</li>
+          <li class="tools">Ruby</li>
+        </ul>
+        <button class="see-project-btn" type="button">See Project</button>
```

Comment:
♻️ Your project is going well so far. Please note though, that when checking the accessibility score, I found this issue with the contrast of most of your buttons. The low-contrast text might make it difficult for some users to read. 

![image](https://user-images.githubusercontent.com/26911718/207352016-40261bc4-a8f1-4518-888c-7397770634de.png)

- [ ] Please double-check the buttons' text color contrast to make it readable.

Reply:
- [ ] Please kindly implement this required change, you might want to use this [tool](https://www.aditus.io/button-contrast-checker/) to balance contrast. 👍 Kindly run your live demo through it and consider making all the changes suggested by the tool.